.golangci.yml

```diff
@@ -1,5 +1,10 @@
 run:
+  # required because the CI/Travis sometimes take to long to run
   timeout: 5m
+
+  # skip files that should not be checked
+  skip-files:
+    - internal/bindata/olm/manifests-0.15.1.go
```

Comment:
Why shouldn't this file be `go fmt`-ed? As I suggested [here](https://github.com/operator-framework/operator-sdk/pull/4062#issuecomment-712562265), we can use a version of `go-bindata` that ensures the output has been formatted correctly.

Reply:
See; https://github.com/operator-framework/operator-sdk/pull/4062#discussion_r508656628

Reply:
All Go files except `vendor/` should be `go fmt`'d.